src/shared/components/search/SearchContainer.tsx

```diff
@@ -104,6 +104,14 @@ export const cannotQueryMessages = {
     'Filtering Align results is not possible as all of its sequences constitute the alignment.',
 };
 
+const reSearchSpecialWords = /(^| )(or|and|not) /gi;
+const rawQueryClean = (raw: string) =>
+  raw
+    // "     app " -> "app"
+    .trim()
```

Comment:
```suggestion
    .trim()
    .replace(/\s+/g, ' ');
```
I don't think multiple spaces would ever need to be preserved in a query?